src/main/java/io/anserini/search/SearchCollection.java

```diff
@@ -238,10 +236,89 @@ public void close() throws IOException {
     }
 
     List<String> queryTokens = AnalyzerUtils.tokenize(analyzer, queryString);
-    RerankerContext context = new RerankerContext<>(searcher, qid, query, queryString, queryTokens, null, args);
+    RerankerContext context = new RerankerContext<>(searcher, qid, query, null, queryString, queryTokens, null, args);
 
     return cascade.run(ScoredDocuments.fromTopDocs(rs, searcher), context);
   }
+  
+  public<K> ScoredDocuments searchNewsBackground(IndexSearcher searcher, K qid, String queryString)
```

Comment:
let's make the terminology consistent?
above you called it "BL"... here it's something different.
I propose "BackgroundLinking"